msg-socket/src/rep/mod.rs

```diff
@@ -35,15 +35,16 @@ pub struct RepOptions {
     max_clients: Option<usize>,
     min_compress_size: usize,
     /// The maximum number of bytes that can be buffered in the write buffer before being flushed.
-    backpressure_boundary: usize,
+    /// This internally sets [`Framed::set_backpressure_boundary`](tokio_util::codec::Framed).
+    write_buffer: usize,
```

Comment:
What do you think of `flush_watermark`? It conveys better that this is soft limit, e.g. if set to 8KiB, you can write a 2MiB message but then it'll try to flush asap.